core_autoloading.txt

```diff
@@ -12,9 +12,13 @@
 
 PHP has had support for class autoloading since PHP 5 and is an extremely useful feature that is relied on to only load classes that are being used within the current request. However, such a feature for function does currently not exist.
 
-The need for such a feature seem very clear as users will create "helper" classes with static methods that are now possible to autoload via the class autoloading mechanism.
+Where programmers who are experienced in other languages would use 'just' a function, most PHP programmers would avoid using a function. For example many PHP programmers write functions as static methods on classes, so that they can take advantage of autoloading, and then use a facade of a marketing term to avoid referring to them as functions.
```

Comment:
```suggestion
Where programmers who are experienced in other languages would use 'just' a function, most PHP programmers would avoid using a function. For example many PHP programmers write functions as static methods on classes, so that they can take advantage of autoloading, and then "use a facade of a marketing term" to avoid referring to them as functions.
```
You might need to clarify what you mean there

Reply:
Or, I could be trolling.

Reply:
Reasonable :p